### DISCUSSION TITLE
Win7 pro 64 unable to connect with engraver

### DISCUSSION THREAD

**Discussion by Matej8888** [arkypita/LaserGRBL#1516] [General]
Hi, I have just purchased Atomstack A5 pro and when connected to a usb port and installed laserGRBL v4.4.1 and installed CH340 driver (all of which successfully), I am still unable to connect my laptop with Atomstack. I'm using Win7pro 64, netFramework 4.8. 
I keep getting the message : 

Under Device Manager/ USB-SERIAL CH340 (COM3)/properties:

"Windows cannot verify the digital signature for the drivers required for this device. A recent hardware or software change might have installed a file that is signed incorrectly or damaged, or that might be malicious software from an unknown source. (Code 52)"

Because of this the laptop does not recognize the Atomstack at all.

Any Ideas what to do?

**Comment by arkypita** [comment]
Where does your driver comes from?

**Reply by Matej8888** [reply]
I have tried directly from the website http:// Atomstack3d.com/a5, but also from the LaserGRBL v4.4.1 software. Same result.

**Reply by Matej8888** [reply]
Any ideas?

**Reply by Matej8888** [reply]
Any words of wisdom, please?

**Reply by gmmanonymus111** [reply]
Well, not a solution, but as a workaround, you might give [this](https://m.majorgeeks.com/content/page/how_to_disable_driver_signature_enforcement.html)  a shot

**Reply by Matej8888** [reply]
gmmanonymus111 : 
I tried it using F8 and selecting the certificate disabling option and it worked. I just have to remember to re-do it every time before using the laser engraver.
I have read about it a little online and it seems like Microsoft wants just money and therefore doesn't release certificates for nothing.

Thanks for the help anyway.
